### ISSUE TITLE
angular-meteor doesn't work with meteor 1.3

### ISSUE THREAD

**Issue by jmwierzbicki** [Urigo/angular-meteor#1277]
I spent few hours digging up through packages, and there is what i found:

Im creating tutorial step 3 app for testing purposes:
unpacking zip 
after that im upgrading meteor and downloading packages:
`meteor update --release METEOR@1.3-rc.2 `

strangely it downloaded angular-meteor 1.3.1 :

root@debian:/opt/testapp2# meteor list
angular                1.3.1* Everything you need to use AngularJS in your Me...
autopublish            1.0.5-rc.2  (For prototyping only) Publish the entire ...
es5-shim               4.5.8-rc.2  Shims and polyfills to improve ECMAScript ...
insecure               1.0.5-rc.2  (For prototyping only) Allow all database ...
jquery                 1.11.6-rc.2  Manipulate the DOM using CSS selectors
meteor-base            1.0.2-rc.2  Packages that every Meteor app needs
mobile-experience      1.0.2-rc.2  Packages for a great mobile user experience
mongo                  1.1.5-rc.2  Adaptor for using MongoDB and Minimongo ov...
session                1.1.3-rc.2  Session variable
standard-minifier-css  1.0.4-rc.2  Standard css minifier used with Meteor app...
standard-minifier-js   1.0.4-rc.2  Standard javascript minifiers used with Me...
tracker                1.0.11-rc.2  Dependency tracker to allow reactive call...

after that i manually updated angular:
`meteor add angular@=1.3.7`

Changes to your project's package version selections:

angular                   upgraded from 1.3.1 to 1.3.7
angular-meteor-data       upgraded from 0.0.8 to 0.3.0
angular-templates*        upgraded from 0.0.3 to 1.0.0
pbastowski:angular-babel  upgraded from 1.0.7 to 1.0.9

from now on, browser throw "angular module not found" basically wall of error: http://pastebin.com/Bk3STyTQ

when downgrading angular to 1.3.1 i got

Changes to your project's package version selections:
angular             downgraded from 1.3.7 to 1.3.1
angular-templates*  downgraded from 1.0.0 to 0.0.3

and browser error was exactly same as on 1.3.7
So i manually downgraded angular-meteor-data down to 0.0.8 and this made it working once again.

**Comment by kamilkisiela** [comment]
Yeah, this is a known error. Should be fixed soon.

**Comment by jmwierzbicki** [comment]
im totally stuck with my project now.
Reinstalled whole meteor back to 1.2.1 and when im trying to meteor update in my project im getting wall of errors... 
`
root@debian:/opt/tcs# meteor remove autopublish blaze-html-templates insecure

Changes to your project's package version selections:

autopublish            removed from your project
blaze-html-templates   removed from your project
caching-compiler       removed from your project
caching-html-compiler  removed from your project
insecure               removed from your project
templating             removed from your project
templating-tools       removed from your project

autopublish: removed dependency
blaze-html-templates: removed dependency
insecure: removed dependency
root@debian:/opt/tcs# meteor add angular angularui:angular-ui-router accounts-password accounts-facebook accounts-twitter accounts-ui-unstyled accounts-ui angularutils:pagination tmeasday:publish-counts cfs:filesystem cfs:standard-packages cfs:ui cfs:s3 dotansimha:accounts-ui-angular u2622:persistent-session meteorhacks:kadira matb33:collection-hooks msavin:mongol brettle:accounts-anonymous brettle:accounts-anonymous-auto brettle:accounts-logout-to-switch meteorhacks:fast-render brettle:accounts-deluxe angular:angular-material jagi:astronomy standard-minifier-css standard-minifier-js angular-meteor-data@=0.0.8
 => Errors while adding packages:

While selecting package versions:
error: Conflict: Constraint underscore@1.0.5-beta.16 is not satisfied by underscore 1.0.4.
Constraints on package "underscore":
* underscore@=1.0.4 <- top level
* underscore@1.0.4 <- meteor-base 1.0.1
* underscore@1.0.4 <- meteor 1.1.11-faster-rebuild.0 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- meteor 1.1.11-faster-rebuild.0 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- webapp 1.2.4-faster-rebuild.0 <- accounts-oauth 1.1.8 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- webapp 1.2.4-faster-rebuild.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- logging 1.0.9-faster-rebuild.0 <- oauth 1.1.6 <- accounts-oauth 1.1.8 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- logging 1.0.9-faster-rebuild.0 <- webapp 1.2.4-faster-rebuild.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- ejson 1.0.7 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- ejson 1.0.7 <- accounts-password 1.1.4
* underscore@1.0.4 <- routepolicy 1.0.6 <- ddp-server 1.2.3-faster-rebuild.0 <- ddp 1.2.2 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- routepolicy 1.0.6 <- ddp-server 1.2.3-faster-rebuild.0 <- ddp 1.2.2 <- accounts-password 1.1.4
* underscore@1.0.4 <- routepolicy 1.0.6 <- meteorhacks:fast-render 2.13.0
* underscore@1.0.4 <- boilerplate-generator 1.0.4 <- webapp 1.2.4-faster-rebuild.0 <- accounts-oauth 1.1.8 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- boilerplate-generator 1.0.4 <- webapp 1.2.4-faster-rebuild.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.4-faster-rebuild.0 <- accounts-oauth 1.1.8 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.4-faster-rebuild.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- blaze-tools 1.0.4 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.4-faster-rebuild.0 <- accounts-oauth 1.1.8 <-
accounts-facebook 1.0.6
* underscore@1.0.4 <- blaze-tools 1.0.4 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.4-faster-rebuild.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- spacebars 1.0.7 <- blaze-html-templates 1.0.1 <- dotansimha:accounts-ui-angular 0.0.2
* underscore@1.0.4 <- blaze 2.1.3 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- blaze 2.1.3 <- cfs:ui 0.1.3
* underscore@1.0.4 <- check 1.1.0 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- check 1.1.0 <- accounts-password 1.1.4
* underscore@1.0.4 <- observe-sequence 1.0.7 <- angular-meteor-data 0.0.8
* underscore@1.0.4 <- id-map 1.0.4 <- binary-heap 1.0.4 <- mongo 1.1.4-faster-rebuild.0
* underscore@1.0.4 <- random 1.0.5 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- random 1.0.5 <- accounts-password 1.1.4
* underscore@1.0.4 <- diff-sequence 1.0.1 <- angular-meteor-data 0.0.8
* underscore@1.0.4 <- webapp-hashing 1.0.5 <- webapp 1.2.4-faster-rebuild.0 <- accounts-oauth 1.1.8 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- webapp-hashing 1.0.5 <- webapp 1.2.4-faster-rebuild.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- ddp-client 1.2.2-faster-rebuild.0 <- ddp 1.2.2 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- ddp-client 1.2.2-faster-rebuild.0 <- ddp 1.2.2 <- accounts-password 1.1.4
* underscore@1.0.4 <- retry 1.0.4 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- retry 1.0.4 <- ddp-common 1.2.2 <- meteorhacks:kadira 2.28.5
* underscore@1.0.4 <- ddp-common 1.2.2 <- ddp-client 1.2.2-faster-rebuild.0 <- ddp 1.2.2 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- ddp-common 1.2.2 <- ddp-client 1.2.2-faster-rebuild.0 <- ddp 1.2.2 <- accounts-password 1.1.4
* underscore@1.0.4 <- ddp-common 1.2.2 <- meteorhacks:kadira 2.28.5
* underscore@1.0.4 <- ddp-server 1.2.3-faster-rebuild.0 <- ddp 1.2.2 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- ddp-server 1.2.3-faster-rebuild.0 <- ddp 1.2.2 <- accounts-password 1.1.4
* underscore@1.0.4 <- callback-hook 1.0.4 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- callback-hook 1.0.4 <- brettle:accounts-anonymous 0.3.1
* underscore@1.0.4 <- minimongo 1.0.10 <- angular-meteor-data 0.0.8
* underscore@1.0.4 <- ordered-dict 1.0.4 <- minimongo 1.0.10 <- angular-meteor-data 0.0.8
* underscore@1.0.4 <- autoupdate 1.2.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- mongo 1.1.4-faster-rebuild.0
* underscore@1.0.4 <- binary-heap 1.0.4 <- mongo 1.1.4-faster-rebuild.0
* underscore@1.0.4 <- http 1.1.2-faster-rebuild.0 <- accounts-twitter 1.0.6
* underscore@1.0.4 <- url 1.0.5 <- http 1.1.2-faster-rebuild.0 <- accounts-twitter 1.0.6
* underscore@1.0.4 <- reload 1.1.4 <- ddp-client 1.2.2-faster-rebuild.0 <- ddp 1.2.2 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- reload 1.1.4 <- ddp-client 1.2.2-faster-rebuild.0 <- ddp 1.2.2 <- accounts-password 1.1.4
* underscore@1.0.4 <- reload 1.1.4 <- hot-code-push 1.0.0 <- meteor-base 1.0.1
* underscore@1.0.4 <- reload 1.1.4 <- session 1.1.1
* underscore@1.0.4 <- session 1.1.1
* underscore@1.0.4 <- reactive-dict 1.1.3 <- meteortoys:toykit 2.2.1 <- msavin:mongol 1.6.2
* underscore@1.0.4 <- reactive-dict 1.1.3 <- session 1.1.1
* underscore@1.0.4 <- minifiers 1.1.8-faster-rebuild.0 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.4-faster-rebuild.0 <- accounts-oauth
1.1.8 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- minifiers 1.1.8-faster-rebuild.0 <- spacebars-compiler 1.0.7 <- boilerplate-generator 1.0.4 <- webapp 1.2.4-faster-rebuild.0 <- meteor-base
1.0.1
* underscore@1.0.4 <- minifiers 1.1.8-faster-rebuild.0 <- standard-minifiers 1.0.3-faster-rebuild.0
* underscore@1.0.1 <- dburles:mongo-collection-instances 0.3.4 <- angular-meteor-data 0.0.8
* underscore@1.0.4 <- caching-html-compiler 1.0.2 <- angular-templates 0.0.3 <- angular 1.3.1
* underscore@1.0.4 <- templating-tools 1.0.0 <- angular-templates 0.0.3 <- angular 1.3.1
* underscore@1.0.4 <- angular-meteor-data 0.0.8
* underscore@1.0.4 <- angular-templates 0.0.3 <- angular 1.3.1
* underscore@1.0.4 <- accounts-password 1.1.4
* underscore@1.0.4 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- rate-limit 1.0.0 <- ddp-rate-limiter 1.0.0 <- accounts-base 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- srp 1.0.4 <- accounts-password 1.1.4
* underscore@1.0.4 <- email 1.0.9-faster-rebuild.0 <- accounts-password 1.1.4
* underscore@1.0.4 <- accounts-oauth 1.1.8 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- oauth 1.1.6 <- accounts-oauth 1.1.8 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- facebook 1.2.2 <- accounts-facebook 1.0.6
* underscore@1.0.4 <- templating 1.1.6-faster-rebuild.0 <- accounts-ui-unstyled 1.1.8
* underscore@1.0.4 <- accounts-twitter 1.0.6
* underscore@1.0.4 <- twitter 1.1.5 <- accounts-twitter 1.0.6
* underscore@1.0.4 <- oauth1 1.1.5 <- twitter 1.1.5 <- accounts-twitter 1.0.6
* underscore@1.0.4 <- accounts-ui-unstyled 1.1.8
* underscore@1.0.4 <- less 2.5.2-faster-rebuild.0 <- accounts-ui 1.1.6
* underscore@1.0.1 <- cfs:base-package 0.0.30 <- cfs:access-point 0.1.49 <- cfs:standard-packages 0.5.9
* underscore@1.0.1 <- cfs:base-package 0.0.30 <- cfs:filesystem 0.1.2
* underscore@1.0.1 <- raix:eventemitter 0.1.3 <- cfs:collection 0.5.5 <- cfs:collection-filters 0.2.4 <- cfs:standard-packages 0.5.9
* underscore@1.0.1 <- raix:eventemitter 0.1.3 <- cfs:collection 0.5.5 <- cfs:standard-packages 0.5.9
* underscore@1.0.1 <- cfs:http-methods 0.0.30 <- cfs:access-point 0.1.49 <- cfs:standard-packages 0.5.9
* underscore@1.0.1 <- cfs:http-publish 0.0.13 <- cfs:access-point 0.1.49 <- cfs:standard-packages 0.5.9
* underscore@1.0.0 <- u2622:persistent-session 0.4.4
* underscore@1.0.4 <- meteorhacks:kadira 2.28.5
* underscore@1.0.3 <- matb33:collection-hooks 0.8.1
* underscore@1.0.3 <- brettle:accounts-anonymous 0.3.1
* underscore@1.0.3 <- brettle:accounts-multiple 0.3.1 <- brettle:accounts-add-service 1.0.0 <- brettle:accounts-deluxe 0.2.2
* underscore@1.0.3 <- brettle:accounts-multiple 0.3.1 <- brettle:accounts-anonymous 0.3.1
* underscore@1.0.3 <- brettle:workaround-issue-4862 0.0.3 <- brettle:accounts-multiple 0.3.1 <- brettle:accounts-add-service 1.0.0 <- brettle:accounts-deluxe 0.2.2
* underscore@1.0.3 <- brettle:workaround-issue-4862 0.0.3 <- brettle:accounts-multiple 0.3.1 <- brettle:accounts-anonymous 0.3.1
* underscore@1.0.3 <- brettle:accounts-logout-to-switch 0.4.2
* underscore@1.0.3 <- brettle:accounts-login-state 0.0.4 <- brettle:accounts-logout-to-switch 0.4.2
* underscore@1.0.0 <- meteorhacks:fast-render 2.13.0
* underscore@1.0.0 <- meteorhacks:picker 1.0.3 <- meteorhacks:fast-render 2.13.0
* underscore@1.0.0 <- meteorhacks:inject-data 2.0.0 <- meteorhacks:fast-render 2.13.0
* underscore@1.0.3 <- brettle:accounts-add-service 1.0.0 <- brettle:accounts-deluxe 0.2.2
* underscore@1.0.3 <- brettle:workaround-issue-4331 0.0.4 <- brettle:accounts-add-service 1.0.0 <- brettle:accounts-deluxe 0.2.2
* underscore@1.0.3 <- brettle:workaround-issue-4970 0.0.4 <- brettle:accounts-add-service 1.0.0 <- brettle:accounts-deluxe 0.2.2
* underscore@1.0.3 <- brettle:accounts-patch-ui 0.1.10 <- brettle:accounts-deluxe 0.2.2
* underscore@1.0.3 <- jagi:astronomy 1.2.10
* underscore@1.0.5-beta.16 <- minifier-css 1.1.8-beta.16 <- standard-minifier-css 1.0.3-beta.16
* underscore@1.0.5-beta.16 <- minifier-js 1.1.8-beta.16 <- standard-minifier-js 1.0.3-beta.16

`
Please, help me - deadline is so close :(

**Comment by kamilkisiela** [comment]
You can try to use angular from npm instead of atmosphere. By angular, I mean angular, not angular-meteor. It should fix the problem.

**Comment by JeremyBYU** [comment]
How do you use angular from npm with Meteor 1.3.  I installed angular from npm and attempt to import it (import angular from angular) and get this error in the console: 
WARNING: Tried to load angular more than once.

**Comment by Furqankhanzada** [comment]
@kamilkisiela i have same issue give me some guide how can i fix this using npm ?

**Comment by danimbrogno** [comment]
@JeremyBYU, @Furqankhanzada  Here is what I have done to install angular-meteor and angular via npm. 

1. Remove the angular meteor package you installed via atmosphere: `meteor remove angular`
2. Make sure you have the ecmascript package installed. If you had the atmosphere angular package installed previously you may have removed it after getting a notice that multiple compilers were trying to handle files with .js extension. So check out `meteor list` and if ecmascript isn't there do:  `meteor add ecmascript`
3. Install angular and angular meteor via npm: `npm install angular angular-meteor --save`
4. Import angular into the script where you setup your main angular module with: `import angular from 'angular';`
5. Import angular-meteor into the script where you setup your main module with: `import 'angular-meteor';`
6. List 'angular-meteor' as a module dependency for your main module: `angular.module('myApp', ['angular-meteor']);`

**Comment by ozsay** [comment]
@danimbrogno Keep in mind that since you use `ecmascript` and not `angular-babel` (which is included in the meteor version of `angular-meteor`, `ng-annotate` is not included in the build process, so you should check your injections and make them work yourself if you want to minify your code.

**Comment by danimbrogno** [comment]
@ozsay good point! Do you know offhand what the impact would be if one were to use angular-babel instead of ecmascript? You would gain annotations, but would some other feature be lost? Is angular-babel compatible with 1.3?

**Comment by JeremyBYU** [comment]
Thanks for your instruction @danimbrogno .  I got it working but I had to also do the following.

1.  meteor remove blaze-html-templates
2. meteor add static-html

One little thing I noticed is that I can see the browser transition from {{greeting}} to "Welcome". What I'm trying to say is that I can visually see angular compile the templates and such, lasts about 1/5 second. Is that normal?

P.S  I also switched out ecmascript pacakge like so

1. meteor remove ecmascript
2. meteor add modules (ecmascript implies modules so you have to add it manually if you remove ecma)
3. meteor add pbastowski:angular-babel

Now ng annotate works fine!

**Comment by jmwierzbicki** [comment]
Is issue with showing angular uncompiled tag while angular loads still case?

**Comment by jamalx31** [comment]
+1

**Comment by riongull** [comment]
@JeremyBYU, I think this might help: 

https://docs.angularjs.org/api/ng/directive/ngCloak

**Comment by JeremyBYU** [comment]
@riongull Yep thats perfect thanks.

**Comment by TWKDreamState** [comment]
I'm having issues with importing and the same warning..

WARNING: Tried to load angular more than once.

I'm thinking that some of the packages (ie. angular-meteor-auth, angularui:angular-ui-router) means that angular is added as a dependency and appears in my versions file. Is there a way around this?

**Comment by jmwierzbicki** [comment]
@riongull Im afraid that ng-cloak does not help there - after updating to meteor 1.3 my applications render white with index.html navigation buttons visible and unstyled - every button is seen, even those hidden by angular for good second, before application initializes - adding or removing fast render makes no difference, there is ng-cloak in my navigation section - doesn't help

**Comment by Urigo** [comment]
Hi everyone.
Can you please try to new `1.3.9_2` version with Meteor 1.3?
also make sure to update to `angular-meteor-auth` version `1.0.1`.
If it's still not working for you, please provide a reproduction that I can check out.
Thanks

**Comment by TWKDreamState** [comment]
@urigo Working perfectly for me from Atmosphere :) Just struggling to get my head around the whole npm thing! Is it best to stick with Atmosphere packages for the time being?

**Comment by jmwierzbicki** [comment]
@Urigo There should be invitation to my repository on Your emai. If You could check out why navigation is shown, before code is compiled. It didn't happened before upgrade to meteor 1.3

**Comment by kamilkisiela** [comment]
@TWKDreamState Feel free to use npm :+1: 

@jmwierzbicki Can you try with latest release which is `1.3.10`?

**Comment by jmwierzbicki** [comment]
still same case:
while it loads: https://i.imgur.com/TGP5dQE.png
after all is complete: https://i.imgur.com/oAtGcOV.png

this is my nav bar code: 
` <md-toolbar flex="10" >
        <div class="md-toolbar-tools">
            <img src="img/thaumcraft_mirror.png" style="width:110px;">
            
            <md-button ui-sref="solver">Helper</md-button>
            <md-button href="/editor-list">Version List</md-button>
            <md-button href="/help">Help</md-button>
            
            <span flex></span>
            <md-button ng-href="/profile" ng-show="$root.isLogged()">profile</md-button>
            <md-button ng-href="/login" ng-hide="$root.isLogged()">Login</md-button>
            <md-button ng-href="/register" ng-hide="$root.isLogged()">Sign up</md-button>

            <md-button ng-href="/logout" ng-show="$root.isLogged()">Logout</md-button>
        </div>
    </md-toolbar> `
list of packages:

`accounts-password                  1.1.8  Password support for accounts
angular                            1.3.10  Everything you need to use AngularJS in your Meteor app
angular:angular-material           1.0.6  Google Material design written for AngularJs (official repo)
angularui:angular-ui-router        0.2.15  angular-ui-router (official): Flexible routing with nested views in AngularJS
brettle:accounts-anonymous         0.3.1  Support anonymous logins
brettle:accounts-anonymous-auto    0.3.5  Automatically create anonymous accounts for each new visitor
brettle:accounts-deluxe            0.2.2  Give all visitors anonymous accounts to which a login service can be added with popular UI packages
brettle:accounts-login-state       0.0.4  Track the current user's login state and register callbacks that help calculate it.
brettle:accounts-logout-to-switch  0.4.2  Requires that signed up users sign out before they sign in to another existing account.
brettle:accounts-patch-ui          0.1.12  Monkey patches accounts UI packages to support logged in users who have not signed up.
cfs:filesystem                     0.1.2  Filesystem storage adapter for CollectionFS
cfs:standard-packages              0.5.9  Filesystem for Meteor, collectionFS
cfs:ui                             0.1.3  CollectionFS, provides UI helpers
es5-shim                           4.5.10  Shims and polyfills to improve ECMAScript 5 support
jagi:astronomy                     1.2.10  Model layer for Meteor
jquery                             1.11.8  Manipulate the DOM using CSS selectors
meteor-base                        1.0.4  Packages that every Meteor app needs
meteorhacks:fast-render            2.14.0  Render your app before the DDP connection even comes alive - magic?
mobile-experience                  1.0.4  Packages for a great mobile user experience
mongo                              1.1.7  Adaptor for using MongoDB and Minimongo over DDP
msavin:mongol                      1.6.2  In-App MongoDB Editor
pbastowski:angular-babel           1.3.4  Babel compiler and ng-annotate for Meteor 1.3
session                            1.1.5  Session variable
standard-minifier-js               1.0.6  Standard javascript minifiers used with Meteor apps by default.
standard-minifiers                 1.0.6  Standard minifiers used with Meteor apps by default.
tracker                            1.0.13  Dependency tracker to allow reactive callbacks
u2622:persistent-session           0.4.4  Persistently store Session data on the client
`


as a reminder, this issue appeared whenever i upgraded to 1.3 , even since RC1, and disappeared when i downgraded meteor to 1.2 again.

**Comment by kamilkisiela** [comment]
@jmwierzbicki  Could you send me some repro of this?

**Comment by kamilkisiela** [comment]
This may happen because Meteor 1.3 puts `<scripts/>` right before `</body>`, so your app's html is being loaded before javascript. Unlike Meteor 1.2 where all scripts have been inside `<head/>`.

**Comment by jmwierzbicki** [comment]
well, is it possible to alter this behavior?

**Comment by kamilkisiela** [comment]
@jmwierzbicki You can use [ngCloak](https://docs.angularjs.org/api/ng/directive/ngCloak). 

---

But I recommend you to create a main component, let's call it `app`.

So if you have something like this:

```html
<body>
  <div ng-controller="MainCtrl"> {{ hello }} </div>
</body>
```

It will become:

```html
<body>
  <app></app>
</body>
```

Where a template of `<app/>` component will be:

```html
<div ng-controller="MainCtrl"> {{ hello }} </div>
```

---

It's always nice to use `angular.bootstrap`, you have [an working example](https://github.com/Urigo/meteor-angular-socially/blob/master/client/main.js) in Socially.

```js
function onReady() {
  angular.bootstrap(document, [
    'your-app-name'
  ], {
    strictDi: true
  });
}

angular.element(document).ready(onReady);
```

This way your whole application won't be showed up until document is fully ready.

**Comment by kamilkisiela** [comment]
About changing this behaviour. Hmm you can check [that PR](https://github.com/meteor/meteor/pull/6375)

**Comment by jmwierzbicki** [comment]
not sure why, but im getting angular.module(...).component is not a function

**Comment by kamilkisiela** [comment]
`component` is available in AngularJS 1.5+

**Comment by jmwierzbicki** [comment]
Thanks, its all working now!
